assets/psalm-phar/composer.json

```
@@ -0,0 +1,9 @@
+{
+	"name": "psalm/phar",
+	"description": "Composer-based Psalm Phar",
+	"license": ["MIT"],
+	"require": {
+		"php": "^7.0"
```

```suggestion
		"php": "^7.1"
```
because main repo requires that.